amundsen_application/static/js/config/config-custom.ts

```diff
@@ -16,6 +16,9 @@ const configCustom: AppConfigCustom = {
     feedbackEnabled: false,
     notificationsEnabled: false,
   },
+  announcements: {
```

Comment:
We don't need to update this file with the defaults, in the public repo it exists as more of an example. So no need to remove it because it does not harm, just clarifying that its not absolutely necessary.